### PR TITLE
chore: updates developer terms

### PR DESCRIPTION
**Pull request by jpvajda**
- This PR updates the developer terms per a request from our legal team. 
- It also adds a reference page to prior terms of service.

### CODE REVIEW

**Review by roadlittledawn — APPROVED** [review]
:shipit:

**Comment by jpvajda** [comment]
we have one final review on this with legal in the AM, so don't merge until I get the confirmation from Jessica Hill.

**Comment by jpvajda** [comment]
just waiting on the final legal approval from Jessica Hill

**Review by clarkmcadoo — APPROVED** [review]

**Review by clarkmcadoo — APPROVED** [review]

**Pull request merged**
